internal/errors/resolver/live_test.go

```diff
@@ -17,59 +17,15 @@ package resolver
 import (
 	"strings"
 	"testing"
-	"time"
 
-	"github.com/GoogleContainerTools/kpt/internal/errors"
 	"github.com/stretchr/testify/assert"
-	"k8s.io/apimachinery/pkg/runtime/schema"
-	"sigs.k8s.io/cli-utils/pkg/apply/taskrunner"
-	"sigs.k8s.io/cli-utils/pkg/kstatus/status"
-	"sigs.k8s.io/cli-utils/pkg/object"
 )
 
 func TestLiveErrorResolver(t *testing.T) {
```

Comment:
Is there a reason to leave this test function in here with all of the test cases removed?

Reply:
I left it here because we really should be adding some more tests here. I have added an additional test now.